.github/workflows/deploy-staging.yml

```
@@ -52,13 +54,15 @@ jobs:
           VITE_APP_ID=${{ secrets.STAGING_VITE_APP_ID }}
           VITE_MEASUREMENT_ID=${{ secrets.STAGING_VITE_MEASUREMENT_ID }}
           EOF
-          )" >> .env.staging
       - name: bun run build
         run: bun run build
+      - name: Rewrite .firebaserc file
+        run: |
+          echo '{"projects": {"default": "one-pager-maker-staging-id"}}' > .firebaserc        
```

yamllint

[error] trailing spaces

(trailing-spaces)